source/api/presentation/2.1/index.md

```diff
@@ -1227,7 +1227,7 @@ collections
 manifests
 : References to manifests contained within the current collection. Each referenced manifest _MUST_ have the appropriate @id, @type and label.
 
-At least one of `collections` and `manifests` _SHOULD_ be present.  Empty collections with neither are allowed but discouraged.
+At least one of `collections` and `manifests` _SHOULD_ be present in the response.  Responses with neither of `collections` or `manifests`, describing collections that have no members, are allowed but discouraged.
 
```

Comment:
"Responses with neither of" -> "Responses without ...". Maybe "i.e.," before "describing collections ... "?